dateutil/test/test_parser.py

```diff
@@ -551,14 +551,20 @@ def testFuzzy(self):
                                   tzinfo=self.brsttz))
 
     def testFuzzyWithTokens(self):
-        s = "Today is 25 of September of 2003, exactly " \
+        s1 = "Today is 25 of September of 2003, exactly " \
             "at 10:49:41 with timezone -03:00."
-        self.assertEqual(parse(s, fuzzy_with_tokens=True),
+        self.assertEqual(parse(s1, fuzzy_with_tokens=True),
                          (datetime(2003, 9, 25, 10, 49, 41,
                                    tzinfo=self.brsttz),
                          ('Today is ', 'of ', ', exactly at ',
                           ' with timezone ', '.')))
 
+        s2 = "http://biz.yahoo.com/ipo/p/600221.html"
```

Comment:
Generally I prefer to break these into separate tests unless there's some compelling reason to do them together. Doesn't really matter, though, so I'll just break it out into two tests next time I'm messing around with the parser tests.

This whole file needs to be parameterized anyway.